packages/webpack.config.js

```diff
@@ -88,21 +88,19 @@ function getConfig(pkg) {
       }),
       new JsonPostProcessPlugin({
         // Add H5Web's R3F dependency to @h5web/lib's `package.json`
-        matchers:
-          pkg === 'lib'
```

Comment:
I realised that I was making R3F an external without adding it as a dependency, so v0.0.21 most likely would not work in `jupyterlab-h5web`.